scripts/run-proxies.sh

```diff
@@ -10,6 +10,13 @@ if [[ -n "${DEBUG:-}" || -n "${CI:-}" ]]; then set -x; fi
 : "${RESTART_MAX_RETRIES:=3}"
 : "${LOCAL_PORT:=}"
 : "${DRY:=}"
+: "${CLUSTER:=$(
+  echo "CLUSTER environment variable is not set. Setting using \`kubectl\` and \`aws sts get-caller-identity\`" >&2
+  kubectx "$(kubectx | grep "$(aws sts get-caller-identity | jq -r '.Account')")" >&2
+  kubectl config current-context | grep -oP '(?<=/).*'
+)}"
```

Comment:
Is this adding a new dependency for devs to run proxies? Then we should update the proxy docs and announce it on slack.

Reply:
Valid point. I'm gonna look into whether I can do this with pure `kubectl`